### PR TITLE
Add ability to specify a custom sorting function for a column

### PR DESCRIPTION
**Pull request by tbroadley**
https://www.pivotaltracker.com/story/show/130121399

### CODE REVIEW

File: js/angular-tablesort.js [hunk 1]
```diff
@@ -269,8 +271,18 @@ tableSortModule.directive('tsWrapper', ['$parse', '$compile', function( $parse,
                 return filteredArr;
             };
 
+            function defaultSort(a, b) {
```
Comment (sankargorthi) [inline js/angular-tablesort.js]: for consistency with the rest of the methods:

``` js
function defaultSort( a, b ) {
```

**Review by sankargorthi — CHANGES_REQUESTED** [review]

File: js/angular-tablesort.js (hunk 1 above)
Comment (sankargorthi) [reply js/angular-tablesort.js]: Same comment for the rest of the new methods.

Comment (tbroadley) [reply js/angular-tablesort.js]: Let me know if I missed any spots in particular.

**Review by sankargorthi — APPROVED** [review]
🕳

File: js/angular-tablesort.js (hunk 1 above)
Comment (sankargorthi) [reply js/angular-tablesort.js]: I think those methods were in the example file.

**Pull request merged**